lib/galaxy/tool_util/verify/interactor.py

```diff
@@ -420,7 +438,16 @@ def run_tool(self, testdef, history_id, resource_parameters={}):
             if isinstance(value, list) and len(value) == 1:
                 inputs_tree[key] = value[0]
 
-        submit_response = self.__submit_tool(history_id, tool_id=testdef.tool_id, tool_input=inputs_tree)
+        submit_response = None
+        for _ in range(30):
```

Comment:
That's kind of a low value on a real server. Maybe allow up to maxseconds ?

Reply:
I've committed that locally and will include it in my next batch of fixes 😆 - I'll merge this while it is green. Thanks for the review!